OpenSim/Simulation/Model/PrescribedForce.cpp

```diff
@@ -42,14 +42,13 @@ using namespace std;
 /**
  * Default constructor.
  */
-PrescribedForce::PrescribedForce(Body* body)
+PrescribedForce::PrescribedForce(const PhysicalFrame* frame)
 {
```

Comment:
This specific constructor performs double duty (as a default constructor if passed in pointer is null) and as actual constructor so practically can't be removed entirely instead replaced by 2 constructors.  For SWIG purposes we can't have a const_ref and const pointer signatures since they collide. Seems outside the scope of this PR.